docs/pipeline/actions/rename-attribute.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "Rename Attribute"
+sidebarTitle: "Rename Attribute"
+---
+
+The "Rename Attribute" Odigos Action can be used to rename attributes from logs, metrics, and traces. Different instrumentations might use different attribute names for similar information. This action let's you to consolidate the names across your cluster.
+
+### Mechanism
+
+The action will remove attributes from the following contexts:
+
+- Resource attributes for spans, metrics and log records.
+- Scope attributes for spans and metrics.
+- Span attributes for spans.
+- Span Events attributes for spans.
+- Metric data point attributes for metrics.
+- Log record attributes for logs.
+
+The action will rename the specified attributes from all telemetry signals that are specified in the `signals` field, regardless of the source, or any other condition.
+
+### Basic Example
+
+The basic example below shows how to rename the deprecated `net.peer.address` attribute to stable `network.peer.address` in all telemetry signals.
+
+Create a file named `rename-net.action.yaml` with the following content:
+
+```yaml
+apiVersion: actions.odigos.io/v1alpha1
+kind: RenameAttribute
+metadata:
+  name: ra-9sz8q
```

Comment:
I think it's better to use a meaningful name here, which will also be aligned with other actions and better practice for the users if they create it manually

Reply:
True that. Thanks!